### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Ceiceiceii**
updated the README file, added a brief overview of our project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,26 @@
...
+ - ✅ UI/UX & Front-end Design
+ - ✅ Front-end and Back-end Connection
+ - ✅ Designed the User Authentication and Login System
+ - ✅ Designed and set up database
+
+Current Working ON and Challenges:
+- [ ] Data scalping on Art Information on Google Culture and Arts
+- [ ] Tried to use Google Map API -> But only have limited budget
+- [ ] Laction Tracking Across Time
+- [ ] Butter Search Function
```
Comment (Nina1o1) [inline README.md:22]: would you mind change it to"Better", "Location"

File: README.md [hunk 2]
```diff
@@ -9,6 +31,12 @@ cd 4-final-project-teambaker/front-end
 npm install
 npm run dev
 ```
+if the is version conflict in  `nmp install` run below command instead
```
Comment (Nina1o1) [inline README.md:34]: "npm install"

**Review by Nina1o1 — COMMENTED** [review]
maybe some changes in wording

**Review by Ceiceiceii — COMMENTED** [review]
Just updated, thx for noticing

**Review by lesleyzhao — APPROVED** [review]

**Pull request merged**